l10n_it_delivery_note/views/stock_delivery_note.xml

```diff
@@ -176,6 +170,13 @@
                                     title="Update to now"
                                 />
                             </div>
+                            <field
+                                name="company_id"
+                                options="{'no_create': True}"
+                                attrs="{'readonly': [('pickings_picker', '!=', [])]}"
+                                groups="base.group_multi_company"
+                                domain="[('id', 'in', context.get('allowed_company_ids', []))]"
```

Comment:
**thought**: in v16 non ho trovato esempi di `context.get('allowed_company_ids', [])`

Tuttavia ho trovato https://github.com/OCA/l10n-italy/blob/16.0/l10n_it_riba/views/configuration_view.xml#L21

Hai verifcato che questo dominio funziona anche su v16? Se si, possiamo lasciarlo così altrimenti lo cambierei come da esempio qui sopra.